src/main/java/inspired/pdf/unbox/base/TableModel.java

```diff
@@ -1,29 +1,38 @@
 package inspired.pdf.unbox.base;
 
-import inspired.pdf.unbox.Align;
-import inspired.pdf.unbox.Bounds;
-import inspired.pdf.unbox.Font;
-import inspired.pdf.unbox.internal.SimpleFont;
-
 import java.util.ArrayList;
 import java.util.Iterator;
+import java.util.LinkedHashMap;
 import java.util.List;
+import java.util.Map;
 import java.util.stream.Collectors;
 
+import inspired.pdf.unbox.Align;
+import inspired.pdf.unbox.Bounds;
+import inspired.pdf.unbox.Font;
+import inspired.pdf.unbox.elements.EmptyCell;
+import inspired.pdf.unbox.elements.TableCell;
+import inspired.pdf.unbox.elements.TextCell;
+
 /**
  * Represents the columns of a layout.
  */
 public class TableModel implements ColumnModel<TableModel.TableColumn> {
 
     public static final float DEFAULT_WIDTH = 1f;
-    public static TableColumn DEFAULT_COLUMN = new TableColumn("", DEFAULT_WIDTH, Align.LEFT, null);
+    public static TableColumn DEFAULT_COLUMN = new TableColumn("", DEFAULT_WIDTH, Align.LEFT, null, null);
 
     private final List<TableColumn> columns = new ArrayList<>();
+    private final Map<Class, TableCell> defaultCells = new LinkedHashMap<>();
+    private final TableCell emptyCell = new EmptyCell();
```

Comment:
Yes, it could be static, I will change that. I wouldn't move it to the TableCell interface though, because I don't see any use-case for it from user-side. It's an implementation detail that we currently use only in a single location. I would suggest that we move it to some package protected location, if we come up with another use for it somewhere else.

Reply:
Ok, I shouldn't post before looking at the whole code. I will make it package protected to remove it from the user facing part of the library.